test/controllers/CanPayWithinSixMonthsControllerSpec.scala

```diff
@@ -73,7 +74,7 @@ class CanPayWithinSixMonthsControllerSpec extends ItSpec {
       stubCommonActions()
       EssttpBackend.Dates.findJourneyExtremeDates(testCrypto, Origins.Epaye.Bta)()
 
-      val result = controller.canPayWithinSixMonths(fakeRequest)
+      val result = controller.canPayWithinSixMonths(Epaye)(fakeRequest)
```

Comment:
since this endpoint is using the common reconstruct session stuff for PEGA you could re-use some of the specs I wrote like this:
https://github.com/hmrc/essttp-frontend/blob/main/test/controllers/PaymentScheduleControllerSpec.scala#L328

Those are error tests around the behaviour of reconstructing.  It'd be nice to get a test where we need to reconstruct the session and it's successful - kind of like this:

https://github.com/hmrc/essttp-frontend/blob/main/test/controllers/PaymentScheduleControllerSpec.scala#L433